ios/ExpoAdapterGoogleSignIn/GoogleSignInAppDelegate.swift

```diff
@@ -1,8 +1,11 @@
-import ExpoModulesCore
 import GoogleSignIn
```

Comment:
maybe this can also be moved into the if body?

Reply:
@giautm wdyt? Thanks 🙂

Reply:
Sorry, Im forget about this. Letme update it now